### PR TITLE
docs: improving contributing.md

### PR DESCRIPTION
**Pull request by himanshu010**
fixes: #1349 
**What kind of change does this PR introduce?**

improving docs
**Did you add tests for your changes?**
N/A
**If relevant, did you update the documentation?**
N/A
**Summary**

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

**Comment by himanshu010** [comment]
@evilebottnawi can you review this

**Comment by evilebottnawi** [comment]
Please fix conflicts

**Comment by snitin315** [comment]
Looks like supressed by #1354

**Comment by himanshu010** [comment]
> Please fix conflicts

@evilebottnawi done👍

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Table of Contents
 -   [Your first Contribution](#your-first-contribution)
 -   [Setup](#setup)
 -   [Running Tests](#running-tests)
+    -   [Using yarn](#using-yarn)
```
Comment (snitin315) [inline .github/CONTRIBUTING.md]:
```suggestion

```

**Comment by evilebottnawi** [comment]
There is no response from the team for a long time (7 days), so I will merge it

**Pull request merged**